packages/circuit-ui/components/Table/Table.tsx

```diff
@@ -60,6 +60,14 @@ export interface TableProps extends HTMLAttributes<HTMLDivElement> {
     currentRows: Row[],
     nextDirection?: Direction,
   ) => Row[];
+  /**
+   * Specifies the intial sort order of the table
+   */
+  sortDirection?: 'ascending' | 'descending';
+  /**
+   * Specifies the row index which sortOrder applies to
+   */
+  sortedRow?: number;
```

Comment:
These props only affect the initial state, updating them after the initial render won't change the table state. May I suggest a small name change to make the purpose clearer?

```suggestion
  /**
   * Specifies the intial sort order of the table
   */
  initialSortDirection?: 'ascending' | 'descending';
  /**
   * Specifies the row index which sortOrder applies to
   */
  initialSortedRow?: number;
```